src/server/modules/gm.ts

```diff
@@ -26,7 +26,24 @@ export class GMToggleGuildAppBanEvent extends ServerSocketEvent implements Serve
     if(!player) return this.notConnected();
     if(player.modTier < ModeratorTier.ChatMod) return this.gameError('Lol no.');
     const result = this.game.gmHelper.guildAppBan(playerName);
-    this.gameMessage(`${playerName} can ${ result ? 'no longer' : 'now' } send guild applications.`);
+    if(typeof result === 'string') this.gameError(result);
```

Comment:
hrmmm. I think we should use _.isString here.

Reply:
Didn't know about that one. Cheers.